src/oas.md

```diff
@@ -117,7 +117,7 @@ Occasionally, non-backwards compatible changes may be made in `minor` versions o
 
 ### Format
 
-An OpenAPI Document that conforms to the OpenAPI Specification is itself a JSON object, which may be represented either in JSON or YAML format.
+An OpenAPI Document that conforms to the OpenAPI Specification is itself a JSON object, which may be represented either in [[RFC8259|JSON]] or [[YAML|YAML]] format.
```

Comment:
Make reference to JSON RFC explicit.

Otherwise our HTML build script would reference the old RFC in

https://github.com/OAI/OpenAPI-Specification/blob/25fc465c31ca10f889d872ea2406172801d34aff/scripts/md2html/md2html.js#L299